qadence_protocols/measurements/protocols.py

```diff
@@ -64,12 +64,12 @@ def verify_robust_shadow(self) -> dict:
         if shadow_groups is None:
             raise KeyError("Shadow protocol requires an option 'shadow_groups' of type 'int'.")
 
-        robust_shadow_correlations = self.options.get("robust_correlations", None)
+        calibration = self.options.get("calibration", None)
```

Comment:
What I meant was that you do:

```python
    if calibration is None:
        calibration = [1.0 / 3.0] * len(bitstring)
```
later in the code. Just wondering if that couldn't be defaulted here but that might not be applicable.

Reply:
Ah not exactly at this stage (as I require the number of qubits which I don't have at the level of dictionaries) but I can move this initialization in the `robust_shadows.py` functions.